ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1121,49 +1142,51 @@ const updateSpeed = function () {
   const DX = 'DX';
   const fast = 'advantage_fast_count';
   const penalty = 'weight_penalty';
-  getAttrs([DX, fast, penalty], function(values) {
+  getAttrs([DX, fast, penalty], function (values) {
     const effective_DX = (Number(values[DX])
-                            + Number(values[fast]));
+      + Number(values[fast]));
     const penalty_n = Number(values[penalty]);
     const DX_reduction = Math.max(0, Math.min(effective_DX,
-      Math.floor(-penalty_n/2)));
-    setAttrs({'speed':
-                Math.max(0, 6 + effective_DX + penalty_n - DX_reduction)});
+      Math.floor(-penalty_n / 2)));
+    setAttrs({
+      'speed':
+        Math.max(0, 6 + effective_DX + penalty_n - DX_reduction)
+    });
   });
 };
 on('change:dx change:advantage_fast_count'
-    + ' change:weight_penalty sheet:opened',
+  + ' change:weight_penalty sheet:opened',
 updateSpeed);
 
 const updateInitiative = function () {
   const IQ = 'IQ';
   const alert = 'advantage_alert_count';
-  getAttrs([IQ, alert], function(values) {
+  getAttrs([IQ, alert], function (values) {
     const effective_IQ = (Number(values[IQ])
-                            + Number(values[alert]));
-    setAttrs({'initiative': effective_IQ});
+      + Number(values[alert]));
+    setAttrs({ 'initiative': effective_IQ });
   });
 };
 on('change:iq change:advantage_alert_count'
-    + ' sheet:opened',
+  + ' sheet:opened',
 updateInitiative);
 
-const updateWeightPenalties = function() {
+const updateWeightPenalties = function () {
   const weight = 'equipment_total_weight';
   const BR = 'BR';
   const packmule = 'advantage_packmule_count';
-  const highlight = function(penalty) {
+  const highlight = function (penalty) {
     return `highlight_weight_penalty_${penalty}`;
   };
-  getAttrs([weight, BR, packmule], function(values) {
+  getAttrs([weight, BR, packmule], function (values) {
     const weight_n = convertToFloat(values[weight]);
     const effective_BR = (Number(values[BR])
-                            + Number(values[packmule]));
+      + Number(values[packmule]));
     let current_level = 0;
     let update = {};
     for (let level = 1; level <= 6; ++level) {
-      const limit = Math.round(((level*level-1)*12/7 + 8.333)
-                                 * 3 * 2**(effective_BR/3));
+      const limit = Math.round(((level * level - 1) * 12 / 7 + 8.333)
+        * 3 * 2 ** (effective_BR / 3));
```

Comment:
I know it is not standard, but I like the coding style I had used here, which lines up the values being summed. I find it much more readable.
I'd like to keep it if there is an easy way to keep lint from getting on the case. Otherwise, I'm happy to dump it.

Reply:
Hmm, what I'd recommend is splitting the operands out into separate values. It'll likely make the code easier to read and understand in isolation.
I'll take a look to see what may help.
Ideally, we let the auto formatting and limiting do their things, since they will help identify common JavaScript bugs.